make.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Run sh with the `-e` flag to automatically exit on a non-zero exit code:
```suggestion
#!/bin/sh -e
```